script.js

```diff
@@ -0,0 +1,31 @@
+function calculate(){
+    var num1 = parseFloat(document.getElementById("number").value);
+    var num2 = parseFloat(document.getElementById("number1").value);
+    var calc = document.getElementById("signs").value;
+    var answer = document.getElementById("result");
+    
+    switch(calc){
+        case "add":
+           answer.value = num1+num2
+            break;
+        case "subtract":
+           answer.value = num1-num2
+            break;
+        case "multiply":
+           answer.value = num1*num2
+            break;
+        case "divide":
+           answer.value = num1/num2
```

Comment:
Put spaces around your operator signs. For example "num1 + num2". It just makes it more readable.